doc/lua_api.txt

```diff
@@ -3974,6 +3974,12 @@ Definition tables
         ^ Node destructor; called after removing node
         ^ Not called for bulk node placement (i.e. schematics and VoxelManip)
         ^ default: nil ]]
+        on_flood = func(pos, oldnode, newnode), --[[
+        ^ Called when a liquid (newnode) is about to flood oldnode.
```

Comment:
Perhaps mention 'floodable = true' here, "only called for non-air nodes with 'floodable = true'".